libs/utils/WordPressUtils/src/test/java/org/wordpress/android/util/LogFileCleanerTest.kt

```diff
@@ -0,0 +1,70 @@
+package org.wordpress.android.util
+
+import android.content.Context
+import android.os.Build
+import androidx.test.core.app.ApplicationProvider
+import java.io.File
+import java.io.FileReader
+import kotlin.random.Random
+import org.junit.After
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+import org.wordpress.android.util.helpers.logfile.LogFileCleaner
+import org.wordpress.android.util.helpers.logfile.LogFileProvider
+
+@RunWith(RobolectricTestRunner::class)
+@Config(sdk = [Build.VERSION_CODES.O_MR1])
+class LogFileCleanerTest {
+    lateinit var logFileProvider: LogFileProvider
```

Comment:
Since the context we are using doesn't seem to change, I think we can use a `val` here instead.

Also, it should be `private` regardless of this change :)

```
private val logFileProvider = LogFileProvider.fromContext(ApplicationProvider.getApplicationContext())
```

Reply:
Fixed the `private` thing, yep. It does change because the `setup` method runs before every test case, so this has to be a `var` so that it can be reinitialized each time.

Reply:
The context won't change though, so the `LogFileProvider` created from it wouldn't change either. Am I missing something? 🤔